frontend/document/credocument.lua

```diff
@@ -71,11 +71,13 @@ local CreDocument = Document:new{
 function CreDocument:zipContentExt(fname)
     local std_out = io.popen("unzip ".."-qql \""..fname.."\"")
     if std_out then
+        local size, ext
         for line in std_out:lines() do
-            local size, ext = string.match(line, "%s+(%d+)%s+.+%.([^.]+)")
-            -- return the extention
-            if size and ext then return string.lower(ext) end
+            size, ext = string.match(line, "%s+(%d+)%s+.+%.([^.]+)")
+            if size and ext then break end
         end
+        std_out:close()
+        return string.lower(ext)
```

Comment:
This isn't doing the same thing as the code you're rewriting in the sense that previously `ext` could never be nil before trying to do `string.lower()`.